src/main/groovy/edu/oregonstate/mist/locations/resources/LocationResource.groovy

```diff
@@ -0,0 +1,42 @@
+package edu.oregonstate.mist.locations.resources
+
+import com.codahale.metrics.annotation.Timed
+import com.google.common.base.Optional
+import edu.oregonstate.mist.locations.core.Location
+import edu.oregonstate.mist.locations.db.LocationDAO
+import io.dropwizard.jersey.params.LongParam
+
+import javax.ws.rs.Consumes
+import javax.ws.rs.GET
+import javax.ws.rs.Path
+import javax.ws.rs.PathParam
+import javax.ws.rs.Produces
+import javax.ws.rs.WebApplicationException
+import javax.ws.rs.core.MediaType
+import javax.ws.rs.core.Response
+
+@Path("/locations")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+class LocationResource {
+    private final LocationDAO locationDAO
+
+    //@todo: is this really needed?
```

Comment:
You are right. I just tested it out and it is required to have this constructor. My todo note was there originally because in groovy you don't need this type of default constructors. I tried removing the constructor and it didn't build. I will remove the comment.